### PR TITLE
check101 (new) prefer IF NOT to NOT ID

### PR DESCRIPTION
**Pull request by AlexandreHT**
Hello, 
This is my first test. I picked an easy case (https://github.com/larshp/abapOpenChecks/issues/722) and it seems to work ok.

Please do not hesitate to review and comment.
BR, 
Alexandre

### CODE REVIEW

**Comment by larshp** [comment]
thanks, guess the text element errors are false positives => linter issue

https://github.com/abaplint/abaplint/issues/1098 created

File: src/checks/zcl_aoc_check_101.clas.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+     <ID>I</ID>
+     <KEY>M02</KEY>
+     <ENTRY>Last statement is CLEAR or FREE</ENTRY>
+     <LENGTH>62</LENGTH>
+    </item>
+    <item>
+     <ID>I</ID>
+     <KEY>M03</KEY>
+     <ENTRY>Last statement is CHECK or EXIT</ENTRY>
+     <LENGTH>62</LENGTH>
```
Comment (g-back) [inline src/checks/zcl_aoc_check_101.clas.xml:33]: Looks like copy paste? M02+M03 are not used in the check and M01 does not correspond to the text in the source

Comment (AlexandreHT) [reply src/checks/zcl_aoc_check_101.clas.xml:33]: My mistake, I work in an FR environment.
Expected text if Prefer IS NOT to NOT IS for M01. M02 and M03 should be deleted.

File: src/checks/zcl_aoc_check_101.clas.xml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<abapGit version="v1.0.0" serializer="LCL_OBJECT_CLAS" serializer_version="v1.0.0">
+ <asx:abap xmlns:asx="http://www.sap.com/abapxml" version="1.0">
+  <asx:values>
+   <VSEOCLASS>
+    <CLSNAME>ZCL_AOC_CHECK_101</CLSNAME>
+    <LANGU>F</LANGU>
```
Comment (larshp) [inline src/checks/zcl_aoc_check_101.clas.xml:7]: please commit the `E`nglish version

**Pull request closed without merging**